versions/3.0.md

```diff
@@ -1945,7 +1945,11 @@ An object to hold data types that can be consumed and produced by operations. Th
 
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="definitionsName"></a>{name} | [Schema Object](#schemaObject) | A single definition, mapping a "name" to the schema it defines.
+<a name="definitionsName"></a>{name} | [Schema Object](#schemaObject) | A single definition, mapping a "name" to the schema it defines.  The following characters are legal in the `definitionsName`: `a-zA-Z0-9.-_()[]\/`
```

Comment:
Hmm, so maybe not ambiguities, but ugliness in the references pointing to definitions with those names.